docs/contracts/accounting-oracle.md

```diff
@@ -3,14 +3,18 @@
 - [Source code](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/AccountingOracle.sol)
 - [Deployed contract](https://etherscan.io/address/0x852deD011285fe67063a08005c71a85690503Cee)
 
+:::note
+Some methods, events and exceptions are inherited from the [BaseOracle](https://github.com/lidofinance/lido-dao/blob/master/contracts/0.8.9/oracle/BaseOracle.sol) contract
+:::
+
 AccountingOracle is a contract where oracles send addresses' balances controlled by the DAO on the Consensus Layer side.
 The balances can go up because of reward accumulation and can go down due to slashing and staking penalties.
 Oracles are assigned by the DAO.
 
 Other major responsibilities of AccountingOracle: updating exited and stuck validators, finalizing withdrawal requests.
 
 Oracle daemons push their reports every frame (225 epochs currently, equal to one day) and when the
-number of the same reports reaches the ['quorum'](#getquorum) value, the report is pushed to the
+number of the same reports reaches the [quorum](#getquorum) value, the report is pushed to the
 [Lido contract][1].
 
 :::note
```

Comment:
It's worth noting that frame length is also not set in stone and can be changed. And if oracle report is delayed it does not extend the report period, unless it's missed. In this case, the next report will have the report period increased.